src/test/java/com/wikia/webdriver/testsuites/testSuite.xml

```diff
@@ -31,7 +31,7 @@
             <class name="com.wikia.webdriver.testcases.adstests.TestAdsKruxIntegration"/>
             <class name="com.wikia.webdriver.testcases.adstests.TestAdsMonitoringOasis"/>
             <class name="com.wikia.webdriver.testcases.adstests.TestAdsNoAdsForSony"/>
-            <class name="com.wikia.webdriver.testcases.adstests.TestAdsOnSpecial"/>
```

Comment:
What happened with this class/file? Is it needed?

Reply:
I renamed it :blush:

Reply:
Oh :/ github didn't show that. Thanks